link/src/main/java/com/stripe/android/link/ui/LinkButtonView.kt

```diff
@@ -40,11 +40,16 @@ import com.stripe.android.link.LinkPaymentLauncher
 import com.stripe.android.link.R
 import com.stripe.android.link.theme.DefaultLinkTheme
 import com.stripe.android.link.theme.linkColors
+import com.stripe.android.uicore.StripeTheme
 
 private val LinkButtonVerticalPadding = 6.dp
 private val LinkButtonHorizontalPadding = 10.dp
-private val LinkButtonShape = RoundedCornerShape(22.dp)
-private val LinkButtonEmailShape = RoundedCornerShape(16.dp) // Button corner radius - padding
+private val LinkButtonShape = RoundedCornerShape(
+    StripeTheme.primaryButtonStyle.shape.cornerRadius.dp
+)
+private val LinkButtonEmailShape = RoundedCornerShape(
+    StripeTheme.primaryButtonStyle.shape.cornerRadius.dp / 2
+)
```

Comment:
Are you sure that this formula won’t resolve in distorted radiuses?

Reply:
Here is what it looks like with a large value:

<img width="325" alt="image" src="https://user-images.githubusercontent.com/99316447/233460948-cb722240-2a38-49c4-baa3-b3444db24193.png">

And here is what it looks like with cornerRadius = 1

<img width="327" alt="image" src="https://user-images.githubusercontent.com/99316447/233460615-7792830d-46c3-406a-b937-24a501204d42.png">

It seems to be ok